src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1694,6 +1789,18 @@ public async Task<JObject> CreateJObjectForVariableValue(SessionId sessionId, Mo
                     ret = await CreateJObjectForNull(sessionId, retDebuggerCmdReader, token);
                     break;
                 }
+                case (ElementType)ValueTypeId.Type:
+                {
+                    var value = retDebuggerCmdReader.ReadInt32();
```

Comment:
Can this become `var _ = retDebuggerCmdReader.ReadInt32();`, since it's not being used?